.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,49 @@
+name: ☝️ Feature request
+description: Suggest an idea for this project
+title: "Feature request: <short description>"
+labels: ["enhancement", "unconfirmed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for suggesting a new idea for this project. Please complete the following to ensure we have all the details to get things started.
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please confirm before submitting a feature request.
+      options:
+        - label: I checked the [documentation](https://docs.fetch.ai/browser-extension-wallet/) and made sure this feature does not already exist.
```

Comment:
https://docs.fetch.ai/browser-extension-wallet/ : 404 - Not found

Reply:
for now we can use this one:
https://docs.fetch.ai/getting-started/how-to-use-browser-wallet/